src/main/io/vtx_common.c

```diff
@@ -0,0 +1,32 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "fc/config.h"
+#include "io/vtx_common.h"
```

Comment:
Presumably should be `drivers/vtx_common.h`

Reply:
The VTX_SETTINGS defines were moved to io/vtx_common.h in an effort to keep the PG declarations with their associated defaults.  They were originally in the io layer, so it felt more appropriate to restore here.  Should I move them back to drivers ?